enterprise-suite/es-grafana/cluster-exporter.json

```diff
@@ -1,7 +1,44 @@
 [
-   {"graphName": "CPU Percent", "promQL":["node_cpu_percent"]},
-   {"graphName": "Memory Available Percent", "promQL":["node_memory_usable_percent"]},
-   {"graphName": "File System Free Percent", "promQL":["node_filesystem_free_percent"]},
-   {"graphName": "Bytes Received Per Second", "promQL":["sum by (instance) (irate(node_network_receive_bytes[1m]))"]},
-   {"graphName": "Bytes Transmitted Per Second", "promQL":["sum by (instance) (irate(node_network_transmit_bytes[1m]))"]}
+   {
+      "graphName": "CPUs Used",
+      "promQL": [
+         "sum by (namespace) (rate(container_cpu_system_seconds_total[1m]) + rate(container_cpu_user_seconds_total[1m]))"
+      ]
+   },
+   {
+      "graphName": "Memory Used",
+      "promQL": [
+         "sum by (namespace) (container_memory_usage_bytes)"
+      ],
+      "yaxes": {
+         "format": "bytes"
+      }
+   },
+   {
+      "graphName": "File System Used",
+      "promQL": [
+         "sum by (namespace) (container_fs_usage_bytes)"
+      ],
+      "yaxes": {
+         "format": "bytes"
+      }
+   },
+   {
+      "graphName": "Bytes Received Per Second",
+      "promQL": [
+         "sum by (namespace) (rate(container_network_receive_bytes_total[1m]))"
+      ],
+      "yaxes": {
+         "format": "bytes"
```

Comment:
Should that be "bytes/s"?

Reply:
the y-axis here is just converting to human readable MB GB kinds of terms. the Per Second is in title